pyproject.toml

```diff
@@ -92,6 +92,7 @@ addopts = """
   --cov=virtual_ecosystem
   --cov-report=html:htmlcov/coverage
   --doctest-modules --ignore=virtual_ecosystem/__main__.py
+  --import-mode importlib
```

Comment:
Out of curiosity, what is this for?

Reply:
Cool! I did not know that.

Reply:
The default is basic the legacy option, but this one is faster and better.